notebooks/notebooks/paper/defense-inference-compute-rob.ipynb

```diff
@@ -0,0 +1,798 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #24.    )
</pre><p>this cell is the same for all three plots I think, maybe move into a function</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlignmentResearch/KataGoVisualizer/pull/125/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>